src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/apiUsage/PrintfScanfArgumentsInspector.java

```diff
@@ -113,7 +124,7 @@ public void visitPhpFunctionCall(@NotNull FunctionReference reference) {
                     final int expectedParametersCount = minimumArgumentsForAnalysis + Math.max(countWithoutPositionSpecifier, maxPositionSpecifier);
 
                     /* check for pattern validity */
-                    final int parametersInPattern = StringUtils.countMatches(content.replace("%%", ""), "%");
+                    final int parametersInPattern = StringUtils.countMatches(content.replace("%%", "").replace("%*",""), "%");
```

Comment:
This is not needed, or?

Reply:
This is required, otherwise count of `parametersInPattern` is to high and `messageParameters` error will be added.